app/gateway-oss/2.4.x/proxy.md

```diff
@@ -695,6 +707,38 @@ Host: ...
 
 [Back to top](#introduction)
 
+#### Normalization behavior
+
+In order to prevent trivial Route match bypass, the incoming request URI from client
+is always normalized according to [RFC 3986](https://tools.ietf.org/html/rfc3986)
+before router match occurs. Specifically, the following normalization techniques are
+used for incoming request URIs which are selected because they generally do not change
```

Comment:
```suggestion
used for incoming request URIs, which are selected because they generally do not change
```

either comma before which clause or change to "which" to "that"

https://www.grammarly.com/blog/which-vs-that/?&utm_source=google&utm_medium=cpc&utm_campaign=10273012991&utm_targetid=dsa-1233402314764&gclid=Cj0KCQjw38-DBhDpARIsADJ3kjkkUiNesJxnbfHpDRYlsmmRt78Mt3lkIElLPs-AhDGLjwOv9otDjL4aAoIVEALw_wcB&gclsrc=aw.ds